framework/filters/Cors.php

```diff
@@ -80,6 +80,10 @@ class Cors extends ActionFilter
      */
     public $actions = [];
     /**
+     * @var bool handle the preflight check and don't proceed with the execution of the request.
+     */
+    public $handleOptions = true;
```

Comment:
Hmm, not sure. allowOptions could indicate that OPTIONS are allowed, and possible to act on with a function. skipOptions doesn't sound right.

IMO handleOptions or processOptions covers what it does, handling the OPTIONS internally instead of passing it trough.